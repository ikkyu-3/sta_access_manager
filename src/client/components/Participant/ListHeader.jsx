// @flow
import * as React from "react";
import styles from "./style.css";

function ListHeader(): React.Node {
  const { header, columnName, columnPurpose, columnStatus, border } = styles;
  return (
    <div className={header}>
      <div className={`${columnName} ${border}`}>名前</div>
      <div className={`${columnPurpose} ${border}`}>目的</div>
      <div className={columnStatus}>入退室</div>
    </div>
  );
}
export default ListHeader;
