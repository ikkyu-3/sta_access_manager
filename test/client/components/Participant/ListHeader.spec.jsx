import React from "react";
import renderer from "react-test-renderer";
import ListHeader from "@components/Participant/ListHeader";

describe("Participant.ListHeader.jsxのテスト", () => {
  describe("スナップショット", () => {
    it("正しいレンダリング", () => {
      const tree = renderer.create(<ListHeader />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
