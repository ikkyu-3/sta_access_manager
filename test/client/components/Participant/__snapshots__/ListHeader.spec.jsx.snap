// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant.ListHeader.jsxのテスト スナップショット 正しいレンダリング 1`] = `
<div
  className="header"
>
  <div
    className="columnName border"
  >
    名前
  </div>
  <div
    className="columnPurpose border"
  >
    目的
  </div>
  <div
    className="columnStatus"
  >
    入退室
  </div>
</div>
`;
